lib/avromatic/model/attribute/union.rb

```diff
@@ -0,0 +1,46 @@
+require 'avromatic/model/attribute_type/union'
+
+module Avromatic
+  module Model
+    module Attribute
+      class Union < Virtus::Attribute
```

Comment:
Probably worth a few class level comments on how this integrates into the Virtus framework.